### DISCUSSION TITLE
Postgres table argo_workflows is empty

### DISCUSSION THREAD

**Discussion by saranyaeu2987** [argoproj/argo-workflows#5748] [Q&A]
Installed Argo using quick start https://raw.githubusercontent.com/argoproj/argo/stable/manifests/quick-start-postgres.yaml
which comes with postgres service. It has 4 tables.

- argo_archived_workflows
- argo_archieved_workflows_labels
- argo_workflows
- schema_history

All tables have data except argo_workflows. Why the workflows/cron workflows deployed to Argo are not persisted in this table?

**Comment by alexec** [comment]
Workflows are only offloaded in they are too large to save as a Kubernetes resource. This is because offloading may fail due to network issue and result in errored workflow. It is common, even preferable, to have this table empty.

**Reply by shawnbyte** [reply]
still not understand，what data is stored in table argo_workflows ?
